### PR TITLE
[RW-1761][risk=no]changes for input number field validation and Title changes for Wizard

### PR DESCRIPTION
**Pull request by smazumd**
changes for input number field validation and Title changes for Wizard

### CODE REVIEW

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.ts [hunk 1]
```diff
@@ -309,4 +309,12 @@ export class ModifierPageComponent implements OnInit, OnDestroy, AfterContentChe
...
     this.dateObjs[index] = new Date(this.form.get(['eventDate', control]).value + 'T08:00:00');
   }
+
+  numberValidation(event) {
+    if (!((event.keyCode > 95 && event.keyCode < 106)
+      || (event.keyCode > 47 && event.keyCode < 58)
+      || event.keyCode === 8)) {
+      return false;
+    }
+  }
```
Comment (dolbeew) [inline ui/src/app/cohort-search/modifier-page/modifier-page.component.ts]: This prevents invalid input but I'm still able to paste in negative values. In that case, there should be an error message similar to the attributes page and the calculate button disabled.

Comment (dolbeew) [reply ui/src/app/cohort-search/modifier-page/modifier-page.component.ts]: Also, we should probably be consistent with how we're doing validation across the app (we're handling it differently in attributes and review) but those need to be cleaned up anyway, especially attributes, so we can address that on a follow-up PR

Comment (smazumd) [reply ui/src/app/cohort-search/modifier-page/modifier-page.component.ts]: Made changes for modifier. No copy paste of negative number will show error message.

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.html [hunk 2]
```diff
@@ -1,4 +1,14 @@
 <form [formGroup]="form">
+  <div *ngIf="showEroor && formChanges" class="alert alert-danger alert-sm">
```
Comment (dolbeew) [inline ui/src/app/cohort-search/modifier-page/modifier-page.component.html]: nit: typo, should be "showError"

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.html [hunk 3]
```diff
@@ -81,7 +91,7 @@
 <div class="container count-preview">
   <div class="row">
     <div class="col-lg-3">
-      <button type="button" [ngClass]="[ preview.get('requesting')? 'greyed-button': 'calculate-btn']"
+      <button type="button" [ngClass]="[ preview.get('requesting') || showEroor? 'greyed-button': 'calculate-btn']"
```
Comment (dolbeew) [inline ui/src/app/cohort-search/modifier-page/modifier-page.component.html]: typo (same as above)

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.ts [hunk 4]
```diff
@@ -110,7 +110,7 @@ export class ModifierPageComponent implements OnInit, OnDestroy, AfterContentChe
 
   dateA = new FormControl();
   dateB = new FormControl();
-
+  showEroor = false;
```
Comment (dolbeew) [inline ui/src/app/cohort-search/modifier-page/modifier-page.component.ts]: typo (same as above)

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.ts [hunk 5]
```diff
@@ -309,4 +308,23 @@ export class ModifierPageComponent implements OnInit, OnDestroy, AfterContentChe
...
+  }
+
+  negativeNumber() {
+   this.modifiers$.forEach(item => {
+    const modArr = item.map(modValue => {
+       return modValue.toJS().operands.map( o => {
+          return o < 0;
+        });
+      });
+     this.showEroor = modArr.toJS().flat().includes(true);
```
Comment (dolbeew) [inline ui/src/app/cohort-search/modifier-page/modifier-page.component.ts]: typo (same as above)

**Pull request merged**